Bot/src/core/config.py

```diff
@@ -0,0 +1,28 @@
+from dotenv import load_dotenv, dotenv_values
+from pathlib import Path
+from pydantic import (BaseSettings, HttpUrl, IPvAnyAddress)
+
+BASE_DIR = Path(__name__).resolve().parent.parent.parent
+# --------------------------------------------------------------------------------- #
+# Getting variables from .env                                                       #
+# --------------------------------------------------------------------------------- #
+load_dotenv()
+
+
+class Settings(BaseSettings):
+    host: str
+    webhook_url: HttpUrl
+    port: int
+    token: str
+    webhook_ip: str
+
+    class Config:
+        fields = {"port": {"env": "bot_port"}, "token": {"env": "telegram_token"}}
+
+
+HOST = (s := Settings()).host
```

Comment:
Предлагаю не использовать эту конструкцию в коде, она только усложняет понимание кода и если посмотреть через дизассемблер на эту конструкцию, то код становится на 3 команды больше и сложнее.
Так код выглядит значительно лучше, не считая очень плохого названия переменной
```suggestion
s = Settings()
HOST = s.host
```
Это очень плохой пример синтаксического сахара, который только ухудшает восприятие кода.
Эта конструкция первая в списке на добавление в документацию как плохие практики